<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>siweidaotu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <widget class="QGraphicsView" name="graph"/>
    </item>
   </layout>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <widget class="QPushButton" name="add_button">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QComboBox" name="selected_node"/>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="update_button">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="delete_button">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="up_button">
       <property name="text">
        <string>Up</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="down_button">
       <property name="text">
        <string>Down</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>graph</zorder>
   <zorder>horizontalLayoutWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1350</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionExport"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>Display</string>
    </property>
    <addaction name="actionDisplay_Boxes"/>
    <addaction name="actionDisplay_Indexes"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionFigure_size"/>
    <addaction name="actionFont_size"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuDisplay"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save as ...</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionDisplay_Boxes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display Boxes</string>
   </property>
  </action>
  <action name="actionDisplay_Indexes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display Indexes</string>
   </property>
  </action>
  <action name="actionFigure_size">
   <property name="text">
    <string>Figure size</string>
   </property>
  </action>
  <action name="actionFont_size">
   <property name="text">
    <string>Font size</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
